### DISCUSSION TITLE
Como funciona el exception Handling para Webflux/ y REST normal bajo el modelo propuesto?

### DISCUSSION THREAD

**Discussion by danielwt1** [bancolombia/scaffold-clean-architecture#398] [General]
Hola, quisiera saber el como ustedes hacen el manejo de excepciones  con un handler para aplicaciones reactivas? ya que pues el handler deberia manejarme las excepciones a nivel de aplicación, lo que intente fue colocar la libreria de webflux en la capa application, porque creia que aca tenia sentido y configurar mi handler tal que 
![image](https://github.com/bancolombia/scaffold-clean-architecture/assets/71347694/2f01d1e4-6df9-463a-94ba-eaa9d20bff98)
tambien configure mi custom attribute tal que 
![image](https://github.com/bancolombia/scaffold-clean-architecture/assets/71347694/bdc78a10-8993-4bb1-9d0f-f6b6c4c2d5e7)
entonces haciendo esto me quedo algo como:
![image](https://github.com/bancolombia/scaffold-clean-architecture/assets/71347694/68dcc028-94c0-4457-9ea0-7f84a95fb15a)
pero al intentar lanzar la excepcion no la toma pese a que debería propagar el error, lo hago con el listener de SQS
![image](https://github.com/bancolombia/scaffold-clean-architecture/assets/71347694/85b32057-d6bf-489c-897e-ec58d70050f2)
estuve revisando todo pero por alguna razon no la captura en handler supongo que es algun problema con la modularidad que se propone en este modelo lo que me hace preguntarme como bajo este modelo se deberian manipular las excepciones a manera global???, otro punto que tambien es interesante en su modelo propuesto las colas FIFO, estan eliminando aunque el servicio falle, lo cual no deberia ocurrir ya que si, se entrego el mensaje pero no se proceso de manera incorrecta pero en el listener esta dando el confirm() pese a que lance un error o devuelva un Mono.error  pese 
![image](https://github.com/bancolombia/scaffold-clean-architecture/assets/71347694/b94b2924-afeb-4154-a3e7-330bcc756652)
exactamente acá es donde da el confirm (), lo he intentado editar haciendo que realmente corte el flujo en este punto pero aun no lo consigo, entonces seria un comportamiento normal?, porque en SQS dice que no debería eliminarse el mensaje si no se proceso dentro del microservicio bien .
@santitigaga

**Comment by oscar-alvarado24** [comment]
Lo pudiste solucionar? y si es así podrías compartir como lo hiciste
